wmn-data.json

```diff
@@ -2300,7 +2300,7 @@
         "uri_check" : "https://github.com/{account}",
         "e_code" : 200,
         "e_string" : "p-nickname vcard-username d-block",
-        "m_string" : "<title>Page not found ",

```

Comment:
Python isn't my language of choice, but trying:
```
import requests

url = "http://github.com/abcdeffsafas"

payload = {}
headers = {
    'Accept-Encoding': 'identity',
}

response = requests.get(url, headers=headers, data=payload)

print(response.text)

```
Returns 'Not Found'

```
print(response.status)
```

Returns 404.

Not sure if this is at all helpful.

Reply:
Try `curl 'https://github.com/abcdeffsafas'   -H 'authority: github.com'   -H 'accept: text/html'` and you get the entire HTML. Without that "text/html" header part, it'll only give the Not Found. Problem is that, if our users are using tools like whatsmyname.app, they will be sending the header data and will get a different result.

This is not that big a deal in reality, since most tools do not use the `m_string` at all.

In the web browser, the response contains `not found` but not `Not Found`. On the command line, it is reversed. So if  a tool does case sensitive matching of the responses, it will get false positives/negatives depending on how the site is checked. 

Since most of the users of this project use the web browser version AND since most tools don't even check the `m_string` I'd like to leave the string as it is.